src/components/tab-bar/tab-bar.tsx

```diff
@@ -100,6 +109,7 @@ export class TabBar {
 
     private setupListeners() {
         this.mdcTabBar.listen(TAB_ACTIVATED_EVENT, this.handleTabActivated);
+        this.scrollArea.addEventListener('scroll', this.handleScroll);
```

Comment:
We should probably make this listener passive
https://developers.google.com/web/updates/2016/06/passive-event-listeners

Reply:
Will do! 👍

Reply:
⚡